src/Task/AbstractTask.php

```diff
@@ -717,11 +718,11 @@ private function validateNice(int $nice = null): bool
 
     private function validatePriority(int $priority): bool
     {
-        if ($priority > 1000) {
+        if ($priority > self::MAX_PRIORITY) {
             return false;
         }
 
-        return $priority >= -1000;
+        return $priority >= self::MIN_PRIORITY;
```

Comment:
What if we use a ternary?

Reply:
how? and how will rector react?

Reply:
Rector has been removed. 

Could be: 

```php
return $priority > self::MAX_PRIORITY ? false : $priority >= self::MIN_PRIORITY;
```

?